__tests__/pages/__snapshots__/index.test.js.snap

```diff
@@ -102,7 +102,7 @@ exports[`IndexPage page renders correctly 1`] = `
         Object {
           "image": "icon-component-mock",
           "title": "Participation Payout",
-          "value": "$22420",
+          "value": "$22781",
```

Comment:
Where is this number taken from? I don't remember last time we changed it, so I would assume it's off by a bit...?

Reply:
Changed 26 days ago (https://github.com/Joystream/joystream-org/pull/141) here

Reply:
The number is taken from 
Sparta (266) + Athens (1310) + Acropolis (16592.63) + Rome (3679.03) + Constantinople (933.63)